tmt-app/src/Components/MajorPage.jsx

```diff
@@ -1,10 +1,30 @@
+import { Container, Typography } from "@mui/material";
 import React from "react";
 import { useParams } from "react-router-dom";
+import { MAJORS_LIST } from "../constants/careerMajorList";
 
 function MajorPage() {
-  const majorChoice = useParams();
+  const params = useParams();
+  let data = [];
 
-  return <div>{majorChoice.major}</div>;
+  const searchData = (target) => {
+    data = MAJORS_LIST[`"${target}"`];
+  };
+
+  searchData(params.major);
```

Comment:
The only way I could think of is to call it directly in DOM. However, I think it might be too messy calling it every time because there are more information we need to display. What are other alternatives to simplify this even more?